### DISCUSSION TITLE
fastmri.org Domain Transfer

### DISCUSSION THREAD

**Discussion by mmuckley** [facebookresearch/fastMRI#293] [Announcements]
### DOMAIN MIGRATION

Hello everyone! We are planning to transfer the domain name at https://fastmri.org/ from Meta to NYU ownership next week.

### WHAT THIS MEANS

**All of the leaderboards will stop working on the date of the transfer.** Current information on the site will be archived (but can be deleted on request). No data will be transferred to NYU. If/when NYU creates a new leaderboard, you will have to create a new account.

The data website at https://fastmri.med.nyu.edu/ will be completely unaffected.

### MITIGATIONS

- **Evaluation on the brain test set:** You can now download the fully-sampled brain test set from https://fastmri.med.nyu.edu/ and run evaluations yourself. The website essentially runs the same code as in [evaluate.py](https://github.com/facebookresearch/fastMRI/blob/main/fastmri/evaluate.py) You may want to log the current leaderboard information if you want to make comparisons to the other models there.
- **Evaluation on knee test set (multicoil and single coil):** This will no longer be possible until NYU rebuilds the leaderboard infrastructure on their version of the website. Instead, you will have to use the knee validation set. You can split the knee validation set into two and use half as "val" for improving your models and the other half as "test".
- Historic leaderboards can be accessed via the archive at https://web.archive.org/web/20230324102125mp_/https://fastmri.org/leaderboards.

### QUESTIONS?

Please post them below.

**Comment by mmuckley** [comment]
This process has finished. Users are now redirected to https://fastmri.med.nyu.edu/.

**Reply by Sarah-2021-scu** [reply]
Hi Mmuckley,

I am trying to test the varnet demo version for knee data by splitting the validation dataset but the test mode does not work. Is there some pr-processing that needs to be done on the validation data to run it in test mode?

**Reply by mmuckley** [reply]
Hello @Sarah-2021-scu, could you create a new discussion thread for this question since it's not about the website?

**Reply by Sarah-2021-scu** [reply]
Thanks for the response. 
I have created a new discussion thread.

**Comment by linyliny** [comment]
How can I access the leaderboard information that you have saved now?

**Reply by mmuckley** [reply]
Hello @linyliny I think you can try to reach out to NYU at [fastmri@med.nyu.edu](https://fastmri.med.nyu.edu/fastmri@med.nyu.edu). We no longer maintain any website infrastructure on the Meta side.

**Comment by youzhaoyirinide** [comment]
Hello，how to see the ranking information of the previous leaderboard？

**Reply by mmuckley** [reply]
Hello @youzhaoyirinide, as I said in the previous message, the only thing I can recommend is to reach out to [fastmri@med.nyu.edu](https://fastmri.med.nyu.edu/fastmri@med.nyu.edu).

**Comment by fsa125** [comment]
Hi @mmuckley, I was trying to download the data from  https://fastmri.org/ and it's saying 502 bad gateway!! Is the data not public now?

1 October, 2023

**Reply by mmuckley** [reply]
Hello @fsa125, I've notified NYU and they're working on getting the site back up.

**Reply by mmuckley** [reply]
Hello @fsa125, the site should be back up.
